gcp/workers/importer/importer_test.py

```diff
@@ -715,7 +715,7 @@ def test_bucket_deletion(self, unused_mock_time: mock.MagicMock,
       imp.run()
     # The schema validation of failures of the files in GCS by
     # _process_deletions_bucket() causes, plus an extra one from the safeguard.
-    self.assertEqual(4, len(logs.output))
+    self.assertEqual(5, len(logs.output))
     self.assertEqual(
         "ERROR:root:Cowardly refusing to delete 2 missing records from GCS for: test",  # pylint: disable=line-too-long
         logs.output[-1])
```

Comment:
I assume this line has to change, since `Cowardly refusing to delete...` is no longer the last log